css/styles.css

```diff
@@ -0,0 +1,299 @@
+HTML{
+    background-color: rgba(45, 46, 46, 0.767);
```

Comment:
Traditionally, we set background for html without opacity, so set it to `#fff` (it's a white). And just copy first 2 selectors (`html, body` and `body`) from css-file from [the project](https://github.com/Dotcommando/adaptive-layout/blob/master/css/styles.css). It's universal selectors.